crates/lang/codegen/src/generator/dispatch.rs

```diff
@@ -807,7 +807,7 @@ impl Dispatch<'_> {
                 }
 
                 impl ::ink_lang::reflect::ExecuteDispatchable for __ink_MessageDecoder {
-                    #[allow(clippy::nonminimal_bool)]
+                    #[allow(clippy::nonminimal_bool, clippy::let_unit_value)]
```

Comment:
When the message returns `()` then the lint would be triggered on this line: https://github.com/paritytech/ink/blob/cc7e4773c47b07336e4aa81d7edf843cbd54b779/crates/lang/codegen/src/generator/dispatch.rs#L742